src/main/webapp/app/guided-tour/guided-tour.component.ts

```diff
@@ -492,7 +492,7 @@ export class GuidedTourComponent implements AfterViewInit, OnDestroy {
      * @return style object for the rectangle beside the highlighted element
      */
     public getOverlayStyle(position: OverlayPosition) {
-        let style;
+        let style = null;
```

Comment:
we would like to avoid the use of `null`

Reply:
This was actually intentional because the method is used in `ngStyle` directive which accepts an object or `null`. I made a change but I had to add `|| null` in the template in case the function returns undefined.